test/integration/controlplane/transformation/transformation_testcase.go

```diff
@@ -172,7 +244,9 @@ func (e *transformTest) getRawSecretFromETCD() ([]byte, error) {
 
 func (e *transformTest) getEncryptionOptions() []string {
 	if e.transformerConfig != "" {
-		return []string{"--encryption-provider-config", path.Join(e.configDir, encryptionConfigFileName)}
+		return []string{
+			"--encryption-provider-config", path.Join(e.configDir, encryptionConfigFileName),
+			"--disable-admission-plugins", "ServiceAccount"}
```

Comment:
we need to disable this because we're creating pods?

Reply:
yes, otherwise the test will end in an error caused by the missing service account:
> error while writing pod: pods "pod1" is forbidden: error looking up service account secret-encryption-test/default: serviceaccount "default"